app/workers/publishing_api_unpublishing_worker.rb

```diff
@@ -0,0 +1,41 @@
+class PublishingApiUnpublishingWorker
+  include Sidekiq::Worker
+
+  def perform(unpublishing_id, locale = I18n.default_locale.to_s)
+    unpublishing = Unpublishing.includes(edition: [:document]).find(unpublishing_id)
+    edition = unpublishing.edition
+    document = edition.document
```

Comment:
Fixed in 74fdc5a. I went for the reducing the DB calls originally but I've benchmarked using `:pluck` and it's ~30% faster. Speed... Woohoo!